config/vendor-api-0.4.0.yml

```diff
@@ -168,8 +189,33 @@ paths:
           $ref: "#/components/responses/Unauthorized"
         "404":
           $ref: "#/components/responses/NotFound"
+  "/test-data/regenerate":
+    post:
+      tags:
+        - Testing
+      summary: Regenerate test data
+      description: Deletes all applications and generates 100 new applications
```

Comment:
not sure but should the description be `Delete ~s~ all applications and generates 100 new applications`?
I've just noticed that the other descriptions do not have the verb in the third person.